src/main/java/ti4/buttons/UnfiledButtonHandlers.java

```diff
@@ -1166,14 +1166,15 @@ public static void revealPOStage(ButtonInteractionEvent event, String buttonID,
                 EndGameService.secondHalfOfGameEnd(event, game, true, true, false);
             } else {
                 var speakerPlayer = game.getSpeaker();
-                ObjectiveHelper.secondHalfOfPeakStage1(game, speakerPlayer, 1);
+                ObjectiveHelper.secondHalfOfPeakStage1(game, speakerPlayer, 1, true);
                 if (!game.isFowMode() && game.getTableTalkChannel() != null) {
                     MessageHelper.sendMessageToChannel(game.getTableTalkChannel(), "## End of Round #" + game.getRound() + " Scoring Info");
                     ListPlayerInfoService.displayerScoringProgression(game, true, game.getTableTalkChannel(), "both");
                 }
-                String message = "The next Objective has been revealed to " + MiscEmojis.SpeakerToken + speakerPlayer.getRepresentationNoPing() + ". When ready, proceed to the Strategy Phase.";
+                String message = "When ready, proceed to the Strategy Phase.";
```

Comment:
Omega Phase grants the Speaker a peek at the upcoming objective after the new objective is revealed. This change prints the reminder to the table in a standalone message, so it isn't deleted on button press.